### DISCUSSION TITLE
Pipewire server not working as audio output

### DISCUSSION THREAD

**Discussion by chicco-carone** [ppy/osu#22138] [Q&A]
### Type

Crash to desktop

### Bug description

I opened the game and in the audio settings i put pipewire sound server to get good latency and in the pw-top command i can see that the game is still using alsa with a backend to pipewire. due to this my config for the quant rate doesn't work because it only applies to pipewire native apps. i tried putting even pulseaudio sound server but it still uses alsa.

[System]
OS:              Arch Linux rolling n/a
Arch:            x86_64
Kernel:          6.1.1-xanmod1-linux-xanmod
Desktop:         KDE
Display Server:  x11

[CPU]
Vendor:          GenuineIntel
Model:           11th Gen Intel(R) Core(TM) i5-11600K @ 3.90GHz
Physical cores:  6
Logical cores:   12

[Memory]
RAM:             15.5 GB
Swap:            4.0 GB

[Graphics]
Vendor:          NVIDIA Corporation
OpenGL Renderer: NVIDIA GeForce RTX 2060/PCIe/SSE2
OpenGL Version:  4.6.0 NVIDIA 525.78.01
OpenGL Core:     4.6.0 NVIDIA 525.78.01
OpenGL ES:       OpenGL ES 3.2 NVIDIA 525.78.01
Vulkan:          Supported

### Screenshots or videos

![Screenshot_20230111_152400](https://user-images.githubusercontent.com/66080458/211830761-bd302422-484c-4fa0-852c-794f49584c31.png)
![Screenshot_20230111_152349](https://user-images.githubusercontent.com/66080458/211830769-0ef7c6a2-5e81-4b18-971e-a4ef72b84143.png)

### Version

2022.1228.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/10393268/runtime.log)

**Comment by chicco-carone** [comment]
The issue is a game issue not crash to desktop

**Comment by huupoke** [comment]
osu!framework is using BASS, which only supports ALSA on Linux. See https://github.com/ppy/osu-framework/issues/2784.

**Comment by chamychu** [comment]
just install this package https://archlinux.org/packages/extra/x86_64/pipewire-alsa/

pacman -S pipewire-alsa

**Reply by chicco-carone** [reply]
Yes I can hear the game but using that package uses the backend alsa and with my pipewire config that doesn't apply because it works only for pipewire app without backends
